src/main/java/edu/kis/vh/nursery/containers/IntArrayStack.java

```diff
@@ -4,39 +4,40 @@
 // jak i implementacji
 
 public class IntArrayStack implements IntContainer {
-    static final int capacity = 12;
-    int total = -1;
-    int[] NUMBERS = new int[capacity];
+    static final int CAPACITY = 12;
+    int TOTAL = -1;
+    Integer[] numbers = new Integer[CAPACITY];
 
     public IntArrayStack() {
     }
 
     public void countIn(int in) {
         if (!isFull())
-            NUMBERS[++total] = in;
+            numbers[++TOTAL] = in;
     }
 
     public boolean callCheck() {
-        return total == empty;
+        return numbers[TOTAL] == null ;
+
     }
 
     public boolean isFull() {
-        return total == 11;
+        return TOTAL == 11;
```

Comment:
NOT FIXED: "PLEASE FIX: Another constant. It is related to CAPACITY (yet, the name is misinforming)"
11 is a constant (not TOTAL)